lib/src/components/NxTable/NxTable.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import {Props} from './types';
+export {Props} from './types';
+
+import './NxTable.scss';
+
+const NxTable = function NxTableElement(props: Props) {
+  const {className, ...attrs} = props;
+  return (
+    <table className={classnames('nx-table', className)} {...attrs} />
+  );
+};
+
+export default NxTable;
```

Comment:
You're missing the `propTypes` definition  — we usually do both `Props` and `propTypes` in case the consuming project isn't using TS.

Reply:
Good catch, I'll added `propTypes`.